ntfs_fr.md

```diff
@@ -370,6 +370,7 @@ stop_time_precision | entier (2) | Optionnel | Précise si l'heure de passage es
         0 (par défaut) - Horaire régulier
         1 - Montée ou descente interdite
         2 - Horaire sur réservation associé à un TAD (si un message est associé au TAD, voir la liaison avec [`comment_links.txt`](#comment_linkstxt-optionnel))
+        3 - Le véhicule ne s'arrête pas (point de passage); dans ce cas, `pickup_type` et `drop_off_type` doivent tous les deux avoir la même valeur 3
```

Comment:
I wasn't there, so I guess there is a reason why having `1` here and `3` in both `pickup_type` and `drop_off_type` doesn't work for this case.

If there is, is it possible to make it explicit ?
Maybe at least tell that `1` is a XOR, but also explain thing more (reply to that comment or better in spec?)

Reply:
The `1` value refers to a generic case where you cannot board or alight  the vehicle. The `3` value is a more specific case where you know you cannot board `nor` alight because the vehicle does not stop. 
The spec refers to both columns (pickup and drop off), so the `1` value can be set to one or the other is clear. The `3` value indicates you must have this value in both fields... Sounds good to me like that :p

Reply:
Right, miss-read the "vehicle doesn't stop".
I was there for the design, only memory failure on my side :stuck_out_tongue:
:heavy_check_mark: So it's fine as-is to me, suggestion below is not mandatory IMO.

Maybe we could add to `1` something like: "Le véhicule s'arrête, mais la montée ou la descente est interdite".

Reply:
I think I will not add the precision you propose because it would mean adding it for all the other values too to be a little consistent. Hope you're fine with it.